mysql-test/suite/galera/r/MW-258.result

```diff
@@ -9,6 +9,7 @@ Variable_name	Value
 wsrep_desync	OFF
 SET SESSION wsrep_sync_wait = 0;
 SET SESSION wsrep_osu_method = RSU;
+SET GLOBAL wsrep_provider_options = 'dbug=d,wsrep_desync_left_local_monitor';
```

Comment:
We should use the script instead
--let $galera_sync_point = apply_monitor_slave_enter_sync
--source include/galera_set_sync_point.inc